programs/nft-voter/src/instructions/create_max_voter_weight_record.rs

```diff
@@ -0,0 +1,57 @@
+use anchor_lang::prelude::*;
+use anchor_spl::token::Mint;
+use spl_governance::state::realm;
+
+use crate::state::MaxVoterWeightRecord;
+
+/// Creates MaxVoterWeightRecord used by spl-gov
+/// This instruction should only be executed once per realm/governing_token_mint to create the account
+#[derive(Accounts)]
+pub struct CreateMaxVoterWeightRecord<'info> {
+    #[account(
+        init,
+        seeds = [ b"max-voter-weight-record".as_ref(),
+                realm.key().as_ref(),
+                realm_governing_token_mint.key().as_ref()],
+        bump,
+        payer = payer
+    )]
+    pub max_voter_weight_record: Account<'info, MaxVoterWeightRecord>,
+
+    /// The program id of the spl-governance program the realm belongs to
+    /// CHECK: Can be any instance of spl-governance and it's not known at the compilation time
+    pub governance_program_id: UncheckedAccount<'info>,
```

Comment:
I have a vague feeling there is some kind of anchor check you can add to make sure this is actually a program pubkey

Reply:
Super cool, Done